WordPress/WordPressTest/PostCoordinatorTests.swift

```diff
@@ -56,16 +56,50 @@ class PostCoordinatorTests: XCTestCase {
 
         expect(post.remoteStatus).toEventually(equal(.pushing))
     }
+
+    func testAutoSavePostThatHasRemote() {
```

Comment:
I'm not sure if the method name describes what's being tested. I initially thought we would be testing `PostCoordinator.autoSave` here since it's `public`. 

How about `testResumeWillAutoSaveUnconfirmedExistingPosts()`? It's wordy but it clarifies the intention of the test. 

Same concern for the `testDraftAndUploadPostWithoutRemote` test method.

Reply:
Done in 7a67f3a2b1c6aa61b954f26adf1f6d20f41aafd2